### PR TITLE
Added More Loading Messages

### PR DESCRIPTION
**Pull request by ScuffedItalian**
I've added all these cool leading messages 😎

- Made in C# becuz C++ will kill me
- The Offical Unoffical Mapper
- C.U.M with the whole family!
- 'Remember Kids
- Chroma Is like drugs It's hella addictive'
- ScuffedItalian was here ;)
- 'Deleting Map hehe
- 'ITS NOT PRONOUNCED GIF!  ITS PRONOUNCED GIF!'

### CODE REVIEW

File: Assets/__Scripts/Localization/Localization.asset [hunk 1]
```diff
@@ -696,3 +696,20 @@ MonoBehaviour:
   - you got games on your phone?
   - i like this one
   - 711nedeaC
+  - Made in C# becuz C++ will kill me
+  - The Offical Unoffical Mapper
```
Comment (XAce1337manX) [inline Assets/__Scripts/Localization/Localization.asset:700]: Are `Official` and `Unofficial` typos intentional?

File: Assets/__Scripts/Localization/Localization.asset [hunk 2]
```diff
@@ -696,3 +696,20 @@ MonoBehaviour:
...
+  - C.U.M with the whole family!
+  - 'Remember Kids
+
+    Chroma Is like drugs
+
+    It''s hella addictive'
+  - ScuffedItalian was here ;)
```
Comment (XAce1337manX) [inline Assets/__Scripts/Localization/Localization.asset:707]: Hmm.

**Review by XAce1337manX — CHANGES_REQUESTED** [review]
* Branch off of dev - not master
* Changes to the scene and sln should not be a part of this PR

File: Assets/__Scripts/Localization/Localization.asset (hunk 1 above)
Comment (ScuffedItalian) [reply Assets/__Scripts/Localization/Localization.asset:700]: If you look inside the loading message suggestion channel, I added these messages word for word. I will fix the typos in my next pr to resolve this error and other errors

**Comment by ScuffedItalian** [comment]
> * Branch off of dev - not master
> 
> * Changes to the scene and sln should not be a part of this PR

Changes to those other 2 files were not aware to me and I'll make sure these files do not get changed in my next pr

**Comment by Top-Cat** [comment]
You can say you'll fix things in the next PR all you like but this ain't getting merged with issues outstanding.

If you're not fixing them here, close the PR

**Pull request closed without merging**